server/src/lib.rs

```diff
@@ -144,18 +138,22 @@ pub fn configure(config: &mut web::ServiceConfig) {
 #[cfg(test)]
 mod test_util {
     use std::sync::Arc;
-    use trustify_graph::graph::{DbStrategy, Graph};
+    use trustify_common::config::Database;
+    use trustify_common::db;
+    use trustify_graph::graph::Graph;
 
     pub async fn bootstrap_system(name: &str) -> Result<Arc<Graph>, anyhow::Error> {
-        Graph::bootstrap(
-            "postgres",
-            "eggs",
-            "localhost",
-            None,
-            name,
-            DbStrategy::External,
+        db::Database::with_external_config(
+            &Database {
+                username: "postgres".to_string(),
+                password: "eggs".to_string(),
+                host: "localhost".to_string(),
+                port: 5432,
```

Comment:
(this part of the port-config is probably related to the github-discussions env/vars etc, not a blocker-to-me :dark_sunglasses: )